src/wallet/wallet2.cpp

```diff
@@ -6297,7 +6297,7 @@ bool wallet2::parse_unsigned_tx_from_str(const std::string &unsigned_tx_st, unsi
     }
     catch (...)
     {
-    LOG_PRINT_L0("Failed to parse data from unsigned tx");
+    LOG_PRINT_L0("Failed to parse decrypted data from unsigned tx");
```

Comment:
This one ought to remain "Failed to parse data from unsigned tx", but line 6622 ought to be "Failed to decrypt signed transaction"